test/lolex-test.js

```diff
@@ -2053,6 +2053,17 @@ describe("lolex", function () {
                 clock.runAll();
                 assert(!called);
             });
+
+            it("passes arguments when installed - GitHub#122", function () {
+                var clock = this.clock = lolex.install();
```

Comment:
@elad-nach - the problem was it also `clock.uninstall()`d, `var clock` is so I don't have to type `this` before it and there is a `afterEach` hook that uninstalls already. The tests above this one show the pattern.